packages/Card/src/components/Avatar/helpers.js

```diff
@@ -0,0 +1,27 @@
+import Tokens from "@paprika/tokens";
+
+export default function calculateAvatarColours(text) {
+  const COLOURS = [
+    { background: Tokens.color.chartColor01, font: Tokens.color.white },
+    { background: Tokens.color.chartColor02, font: Tokens.color.white },
+    { background: Tokens.color.chartColor03, font: Tokens.color.black },
+    { background: Tokens.color.chartColor04, font: Tokens.color.black },
+    { background: Tokens.color.chartColor05, font: Tokens.color.white },
+    { background: Tokens.color.chartColor06, font: Tokens.color.white },
+    { background: Tokens.color.chartColor07, font: Tokens.color.black },
+    { background: Tokens.color.chartColor08, font: Tokens.color.white },
+    { background: Tokens.color.chartColor09, font: Tokens.color.white },
+    { background: Tokens.color.chartColor10, font: Tokens.color.white },
+    { background: Tokens.color.chartColor11, font: Tokens.color.black },
+    { background: Tokens.color.chartColor12, font: Tokens.color.black },
+  ];
```

Comment:
perhaps use some consts?
```
const tokenColours = Tokens.color
const black = tokenColours.black
const white = tokenColours.white
const COLOURS = [
    { background: tokenColours.chartColor01, font: white },
    { background: tokenColours.chartColor02, font: white },
    { background: tokenColours.chartColor03, font: black },
    { background: tokenColours.chartColor04, font: black },
    { background: tokenColours.chartColor05, font: white },
    { background: tokenColours.chartColor06, font: white },
    { background: tokenColours.chartColor07, font: black },
    { background: tokenColours.chartColor08, font: white },
    { background: tokenColours.chartColor09, font: white },
    { background: tokenColours.chartColor10, font: white },
    { background: tokenColours.chartColor11, font: black },
    { background: tokenColours.chartColor12, font: black },
  ];
```

Reply:
Nah i disagree

Reply:
Why? Readability?

Reply:
Yeah i think its easy enough to read, all fits nicely with prettier.

Reply:
🤩